wire-ios/Wire-iOS Tests/DateFormatterTests.swift

```diff
@@ -129,25 +129,18 @@ final class DateFormatterTests: XCTestCase {
         // GIVEN
         let twoHourBefore = Calendar.current.date(byAdding: .hour, value: -2, to: Date())!
```

Comment:
I think we should set a fixed time.
We can also test several values, like shortly before/after midnight or noon.

Reply:
Mm I thought about but how do we when then deal with the fact that we format the time differently depending on if the date/time is from yesterday, 5 minutes ago or two hours ago.

I suppose we would then also need to set the **now** time to some fixed value.

Reply:
I see.